### DISCUSSION TITLE
Questio on file download location.

### DISCUSSION THREAD

**Discussion by ahk52899** [OWASP/masvs#640] [Q&A]
Where can I download this app? I want to learn how to use it however I can't find a download link. I have been trying for a couple hours and can't find it. Please someone send me download link for my kali virtual machine and windows virtual machine. Is there also a step-by-step guide on how to use this app? I saw the guide that was posted but I looked at it from what I can tell it just gives us info about it not a step by step guide on how to use it.

**Comment by cpholguera** [comment]
Hello @ahk52899, are you looking for the MSTG crackmes apps? 

https://github.com/OWASP/owasp-mstg/tree/master/Crackmes

**Reply by ahk52899** [reply]
> Hello @ahk52899, are you looking for the MSTG crackmes apps?
> 
> https://github.com/OWASP/owasp-mstg/tree/master/Crackmes

I believe so - my company and I want to use OWASP's MAVS tool to enhance the security of our mobile app. Where is the download link to the tool? ALso is there an instruction manual in how to learn how to use the app.?

**Reply by cpholguera** [reply]
Hi @ahk52899, this is a nice intro to the project and gives you a lot of context (it even includes our latest updates): https://youtu.be/qddf7MSpdK4

I hope that helps you getting started. Feel free to reach out again for any questions you may have. Have a great weekend!

**Reply by ahk52899** [reply]
> Hi @ahk52899, this is a nice intro to the project and gives you a lot of context (it even includes our latest updates): https://youtu.be/qddf7MSpdK4
> 
> I hope that helps you getting started. Feel free to reach out again for any questions you may have. Have a great weekend!

Thanks. How do I download the OWASP-MASVS app? or was that the first link above? Or is There no app?

**Reply by cpholguera** [reply]
@ahk52899 The MASVS is a standard for testing the security of mobile apps. The MSTG contains the information on how to perform the tests. We also have a couple of apps that you can use as samples for testing or simply for learning while trying to "crack" them. They're called "Crackmes" and they are in the first link I sent you.

**Reply by ahk52899** [reply]
> @ahk52899 The MASVS is a standard for testing the security of mobile apps. The MSTG contains the information on how to perform the tests. We also have a couple of apps that you can use as samples for testing or simply for learning while trying to "crack" them. They're called "Crackmes" and they are in the first link I sent you.

Oh alright thanks. Do you have a manual in how to use the crackme's and install them?

**Reply by cpholguera** [reply]
You can check the Crackmes README where you can find how to install them and some initial instructions.

https://github.com/OWASP/owasp-mstg/blob/362cc0c9633961c88362bceee3fcfca5a6cdd175/Crackmes/README.md

**Reply by ahk52899** [reply]
> You can check the Crackmes README where you can find how to install them and some initial instructions.
> 
> https://github.com/OWASP/owasp-mstg/blob/362cc0c9633961c88362bceee3fcfca5a6cdd175/Crackmes/README.md

Is there a free emulator I can use for the tools?

**Reply by cpholguera** [reply]
Yes, you can install Android Studio on your computer and it comes with a free Android Emulator. You can install APK files and run them in the emulator.

For iOS, it's not possible to install the provided IPA files in the Xcode emulator. You need to have an Apple device (for example an iPhone or iPad).

**Reply by ahk52899** [reply]
> Yes, you can install Android Studio on your computer and it comes with a free Android Emulator. You can install APK files and run them in the emulator.
> 
> For iOS, it's not possible to install the provided IPA files in the Xcode emulator. You need to have an Apple device (for example an iPhone or iPad).

I have an Ipad and an I-Phone 12. In the readme instructions does it say how to install and use the app to do penetration tests on my company's mobile app? Also would this do anything to corrupt my IPAD or I-Phone? I'm also having trouble installing OWASP ZAP on my windows 10 host machine. I have java installed. But OWASP isn't sensing it is installed. WHy is this? What is the fix?

**Reply by cpholguera** [reply]
Regarding OWASP ZAP, I recommend you to contact them directly: https://groups.google.com/g/zaproxy-users

---

The Crackme Apps are not tools to perform penetration tests, they are challenges for you to practice what you've learned after reading the OWASP MSTG (Mobile Security Testing Guide).

Here's the online version:

https://mobile-security.gitbook.io/mobile-security-testing-guide/


And the PDF version:

https://github.com/OWASP/owasp-mstg/releases/download/v1.4.0/OWASP_MSTG-v1.4.0.pdf


If you're new to Mobile Security I recommend you to read the MSTG and try to solve the level 1 crackmes which doesn't require any installation and can be solved by unzipping the app and analyzing it. Once you've learned how to install apps and more you can continue practicing.
